src/mame/misc/neomania.cpp

```diff
@@ -16,44 +16,42 @@ The "NEO MANIA ADAPTER BOARD" contains:
     JMP3 (three positions) - Unknown function
    2 x Coin acceptors ports
    1 x Bank of 8 dipswitches (unknown function)
+
+C:\Neomania folder contains ppm.exe, which is the driver for the parallel port device.
+It also contains a password protected "Guard.zip", copy protection?
+C:\Windows has driver installs for a PCI Sound Blaster and an ATI mach 64 / Bt829 derivative.
+
+
+TODO:
+- HDD image doesn't boot in neither shutms11 nor pcipc, mangled MBR boot record;
+- Extract "Guard.zip" and understand what is for;
```

Comment:
Does it use a custom BIOS to boot the hard disk without a standard MBR?  Given the BIOS hasn’t been dumped, all bets are off.

Reply:
I'll eventually try to use a recover disk but I'm kinda positive this being a lost cause until more info about the MB is given back.

Reply:
Also iirc I can't use an English system recovery disk over a Portuguese Windows 98, it can't simply work.
So yeah, lost cause.